form.html

```diff
@@ -5,56 +5,119 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Form Dipendenti</title>
 </head>
+
+<style>
+    [data-tooltip]{ /*[data-tooltip] prende l'attributo data-tooltip e ne gestisce lo stile*/         
```

Comment:
piccola nota, qui non prendi l'attributo in sè ma gli elementi che hanno quell'attributo